src/components/modeler/Modeler.vue

```diff
@@ -40,6 +40,7 @@
       </b-col>
 
       <mini-paper
+        v-if="miniMapOpen"
```

Comment:
Not rendering the mini paper (as it's closed by default on load anyway) reduced the load time by 2-3x.